dist-unminified/recorder.js

```diff
@@ -226,6 +232,26 @@ Recorder.prototype.resume = function() {
   }
 };
 
+Recorder.prototype.muteMicrophone = function() {
```

Comment:
These seem redundant as it is the same as setting gain to 0.  I prefer to have an interface that do not offer multiple ways to do the same thing.

Reply:
No worries, I've removed the `muteMicrophone()`, `unmuteMicrophone()`, & their callbacks